.evergreen/config_generator/etc/distros.py

```diff
@@ -80,6 +80,8 @@ def validate_os_ver(cls, value):
     Distro(name='rhel80-small', os='rhel', os_type='linux', os_ver='8.0', size='small'),
     Distro(name='rhel84-large', os='rhel', os_type='linux', os_ver='8.4', size='large'),
     Distro(name='rhel84-small', os='rhel', os_type='linux', os_ver='8.4', size='small'),
+    Distro(name='rhel8.7-large', os='rhel', os_type='linux', os_ver='8.7', size='large'),
+    Distro(name='rhel8.7-small', os='rhel', os_type='linux', os_ver='8.7', size='small'),
```

Comment:
```suggestion
    Distro(name='rhel87-large', os='rhel', os_type='linux', os_ver='8.7', size='large'),
    Distro(name='rhel87-small', os='rhel', os_type='linux', os_ver='8.7', size='small'),
```
Suggest omitting `.` for consistency with other RHEL distro names.